packages/kit/src/core/adapt/prerender.js

```diff
@@ -199,6 +205,9 @@ export async function prerender({ cwd, out, log, config, build_data, fallback, a
 					const attrs = match[2];
 
 					if (element === 'a' || element === 'link') {
+						const rel = get_rel(attrs);
+						if (rel === 'external') continue;
```

Comment:
this should probably check if it contains external rather than equaling external in case there are multiple values set in `rel`

Reply:
Yes, you are right, i haven't thought about multiple values. Maybe it's better to check only `external` value in function, because we don't need any other values in `rel`?

Reply:
sure. that sounds reasonable